test-integration/helpers/scores.coffee

```diff
@@ -48,11 +50,14 @@ class Scores extends TestHelper
     @el.ccScoresLink.click()
     @waitUntilLoaded()
 
-  doneGenerating: =>
-    #@test.driver.wait =>
-      #@test.driver.isElementPresent(css: @el.downloadExport)
+  downloadExport: =>
+    @test.sleep(5000, 'waiting for export')
```

Comment:
Can you add a class to the `iframe` when downloading is complete (& then clear it) instead of waiting for a specific period of time?